### PR TITLE
Fix docs

### PR DESCRIPTION
**Pull request by luizcmarin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,13 +27,10 @@
 
 ## Installation
 
-If you do not have [Composer](https://getcomposer.org/), you may install it by following the instructions
-at [getcomposer.org](https://getcomposer.org/doc/00-intro.md).
-
-You can then install this project template using the following command:
+The package could be installed with [Composer](https://getcomposer.org):
```
Comment (vjik) [inline README.md:30]: This is not package. It's good as is.

File: README.md [hunk 2]
```diff
@@ -51,31 +48,39 @@ The application template has the following structure:
 
 ```
 config/             Configuration files.
+    common/
+    console/
+    environments/
+    web/
```
Comment (vjik) [inline README.md:54]: It's good as is.

File: README.md [hunk 3]
```diff
@@ -51,31 +48,39 @@ The application template has the following structure:
...
 docs/               Documentation.
 public/             Files publically accessible from the Internet.
     assets/         Published assets.
     index.php       Entry script.
 resources/          Application resources.
     assets/         Asset bundle resources.
     layout/         Layout view templates.
-    view/           View templates.
+    messages/
```
Comment (vjik) [inline README.md:62]: Add descriptions for new folders.

File: README.md [hunk 4]
```diff
@@ -193,32 +198,16 @@ See ["Logging"](https://github.com/yiisoft/docs/blob/master/guide/en/runtime/log
 ],
 ```
 
-## Testing
```
Comment (vjik) [inline README.md:196]: Not need extract to internals in templates.

File: composer.json [hunk 5]
```diff
@@ -6,6 +6,26 @@
         "yii3",
         "app"
     ],
+    "homepage": "https://github.com/yiisoft/app/",
```
Comment (vjik) [inline composer.json:9]: Not need in template

**Review by vjik — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (luizcmarin) [reply README.md:30]: ok

File: README.md (hunk 3 above)
Comment (luizcmarin) [reply README.md:62]: I don't know what description you like...

**Review by vjik — APPROVED** [review]

**Pull request merged**